docs/manual/src/tutorial/foreign_language_bindings.md

```diff
@@ -8,9 +8,9 @@ The next step is to have UniFFI generate source code for your foreign language.
 
 First, make sure you have installed all the [prerequisites](./Prerequisites.md).
 
-Ideally you would then run the `uniffi-bindgen` binary from the `uniffi` crate to generate your bindings.  However, this
```

Comment:
These changes and below seem unrelated to this specific PR?
Intentionally?
Fine with me to land them along if it is.